Marlin/src/lcd/menu/menu_tune.cpp

```diff
@@ -195,7 +199,8 @@ void menu_tune() {
   //
   #if ENABLED(FT_MOTION_MENU)
     void menu_tune_ft_motion();
-    SUBMENU(MSG_FIXED_TIME_MOTION, menu_tune_ft_motion);
+    if(ftMotion.cfg.active)
```

Comment:
Always put a space between a statement like `if` or `while` and the opening parenthesis.

Reply:
This menu item should always be available, even if FT Motion is disabled, if it contains menu items for configuring FT Motion.